harness/determined/cli/user_groups.py

```diff
@@ -0,0 +1,229 @@
+    resp = bindings.post_CreateGroup(session, body=body)
+    group = resp.group
+
+    print(f"user group with name {group.name} and ID {group.groupId} created")
+    if group.users:
+        print(f"{', '.join([g.username for g in group.users])} was added to the group")
+
+
+@authentication.required
+def list_groups(args: Namespace) -> None:
+    body = bindings.v1GroupSearchRequest(offset=args.offset, limit=args.limit)
+    resp = bindings.post_GetGroups(setup_session(args), body=body)
+    if args.json:
+        print(json.dumps(resp.to_json(), indent=2))
+    else:
+        if resp.groups is None:
+            resp.groups = []
+        render.render_objects(
+            v1GroupHeaders, [render.unmarshal(v1GroupHeaders, g.to_json()) for g in resp.groups]
+        )
+
+
+@authentication.required
+def describe_group(args: Namespace) -> None:
+    session = setup_session(args)
+    group_id = group_name_to_group_id(session, args.group_name)
+    resp = bindings.get_GetGroup(session, groupId=group_id)
+    group_details = resp.group
+
+    if args.json:
+        print(json.dumps(group_details.to_json(), indent=2))
+    else:
+        print(f"group ID {group_details.groupId} group name {group_details.name} with users added")
+        if group_details.users is None:
+            group_details.users = []
+        render.render_objects(
+            v1UserHeaders,
+            [render.unmarshal(v1UserHeaders, u.to_json()) for u in group_details.users],
+        )
+
+
+@authentication.required
+def add_user_to_group(args: Namespace) -> None:
+    session = setup_session(args)
+    usernames = args.usernames.split(",")
+    group_id = group_name_to_group_id(session, args.group_name)
+    user_ids = usernames_to_user_ids(session, usernames)
+
+    body = bindings.v1UpdateGroupRequest(groupId=group_id, addUsers=user_ids)
+    resp = bindings.put_UpdateGroup(session, groupId=group_id, body=body)
+
+    print(f"user group with ID {resp.group.groupId} name {resp.group.name}")
+    for user_id, username in zip(user_ids, usernames):
+        print(f"user added to group with username {username} and ID {user_id}")
+
+
+@authentication.required
+def remove_user_from_group(args: Namespace) -> None:
+    session = setup_session(args)
+    usernames = args.usernames.split(",")
+    group_id = group_name_to_group_id(session, args.group_name)
+    user_ids = usernames_to_user_ids(session, usernames)
+
+    body = bindings.v1UpdateGroupRequest(groupId=group_id, removeUsers=user_ids)
+    resp = bindings.put_UpdateGroup(setup_session(args), groupId=group_id, body=body)
+
+    print(f"user group with ID {resp.group.groupId} name {resp.group.name}")
+    for user_id, username in zip(user_ids, usernames):
+        print(f"user removed from the group with username {username} and ID {user_id}")
+
+
+@authentication.required
+def change_group_name(args: Namespace) -> None:
+    session = setup_session(args)
```

Comment:
check that `new_group_name` can't be empty

Reply:
```new_group_name``` is also a positional arg